chicken-sloths/bangazon-api-sprint1
# Standardized File Names

## There are merge conflicts in this! I'll deal with them at the end!

# Description
This PR changes a bunch of file names so they look like they were written by the Great Chicken Sloth in the Sky rather than a bunch of chicken sloths on the mortal plane.

## Number of Fixes
Fixes issue #114 and #96

## Expected Behavior
Naming conventions for file names:

### Generates JSON
- Camelcase
- Plural with underscore
- `/data/faker/products/_productsFaker.js`

### Route
- Camelcase
- Plural plus 'Router'
`/routes/products/productsRouter.js`

### Model
- PascalCase
- Plural
`/models/products/ProductsModel.js`

### Controller
- camelCase
- Plural
`/controllers/products/productsCtrl.js`

## Steps to Test Solution

1. `` npm data: generate`` shouldn't throw any errors (I redid the file names for faker generators)
1. `` npm start``
1.  Run gets on all the routes to make sure everything still works
1. `/employees`
1. `/departments`
1. `/training-programs`
1. `/computers`
1. `/customers`
1. `/products`
1. `/orders`
1. `/payment-options`
1. `/product-types`
